### PR TITLE
Feature: improve claim management

### PR DESCRIPTION
**Pull request by Aurelien30000**
# This PR aims to improve claim system and reduce impact of multi sync commands on the server tick rate and gameplay.

- Introduce MultiTaskManager to spread load of huge operations.
- Move some classes to a more appropriate package (api mainly).
- Rework some methods to optimize them.
- Optimize block location lookups in chunks getting rid of ``Chunk#getBlock#getLocation`` method.
- Send particles async _(sending packets is safe)_.
- Remove all bad wildcard imports.
- Remove all local final fields for a clean codebase.

- Overall, the following commands have been greatly improved, they are now only able to process 10 chunks/lands per **sync** tick, one command after another:
```
/land multiclaim
/land multiunclaim
/land unclaimall
/land clear
```
Spark report under a "spam pressure" of ``/land multiclaim RECTANGULAR 8`` _(unclaiming is rather fast)_ https://spark.lucko.me/TGwb1vnInp ~10% max of ticks on my computer

### CODE REVIEW

**Review by SirYwell — CHANGES_REQUESTED** [review]
There are some wildcard imports, please revert them to fully qualified imports.

Side note: I personally don't like the usage of `final` besides from fields - not sure if we want to have them in the code base, but I don't care that much about it.

**Comment by RainbowDashLabs** [comment]
> There are some wildcard imports, please revert them to fully qualified imports.
> 
> Side note: I personally don't like the usage of `final` besides from fields - not sure if we want to have them in the code base, but I don't care that much about it.

Regarding the final fields:
My current inspections would remove them again on an global formatting. I would prefer too not introduce them.
Also since we don't use them in the codebase and I also think that is is mostly redundant and not needed at all.
Its mostly overhead and nearly everyone does not use final on local fields

**Comment by Aurelien30000** [comment]
Done 🚀

File: LandLord-legacy/src/main/java/biz/princeps/landlord/manager/MobManager.java [hunk 1]
```diff
@@ -86,9 +86,9 @@ public IMob valueOf(String name) {
 
     public static class Mob implements IMob {
 
-        final EntityType t;
-        final int egg;
-        final String permission;
+        EntityType t;
+        int egg;
+        String permission;
```
Comment (SirYwell) [inline LandLord-legacy/src/main/java/biz/princeps/landlord/manager/MobManager.java:91]: Those are fields, that's okay to be final

Comment (Aurelien30000) [reply LandLord-legacy/src/main/java/biz/princeps/landlord/manager/MobManager.java:91]: Nuked them without considering such variables, ty!

**Review by SirYwell — DISMISSED** [review]

**Review by RainbowDashLabs — APPROVED** [review]

**Review by RainbowDashLabs — APPROVED** [review]

**Review by SirYwell — APPROVED** [review]

**Pull request merged**